core/runtime/src/main/java/io/quarkus/runtime/ThreadsConfig.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * The core thread pool config. This thread pool is responsible for running
+ * all blocking tasks.
+ */
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class ThreadsConfig {
```

Comment:
We should probably call this "ThreadPoolConfig".  (There are other threads in the app that won't be governed by this config).